### DISCUSSION TITLE
Landing page (index.php)

### DISCUSSION THREAD

**Discussion by oh1jw** [magicbug/Cloudlog#1045] [Ideas / Feature Requests]
Ability to edit the "landing page info" for index.php for non registered viewers.
Like putting some info about your station or upcoming activations or your full life story with every bits and bobs.
Or totally remove the qso infos etc..

(idea featured and moved from here: https://github.com/magicbug/Cloudlog/discussions/789#discussioncomment-787215)

**Comment by dfbarnes** [comment]
Personally, I don't think the landing page info options need to be very robust, just simple.

Option in settings to keep the existing map:  "Display map on landing page, Y/N"

Option in settings to keep the existing log information: "Display log information on landing page, Y/N"

Option in settings to "Add custom greeting to landing page" (text only) "Y/N" and text field.

Option in settings for "Add custom link(s) to landing page" with link title and URLs.

I think this would give visitors a very brief bit of info, and we could then use links to point them to places outside Cloudlog for additional info (all the life story, photos, etc.) like QRZ, facebook, twitter, etc.

This would keep things simple and avoid trying to turn Cloudlog's landing page into a mini-CMS.

**Comment by M0TZO** [comment]
I'm not sure about a landing page. For me somewhere to post a link to my blog or QRZ page would be enough.
